optaplanner-docs/src/modules/ROOT/pages/release-notes/release-notes.adoc

```diff
@@ -0,0 +1,241 @@
+[[releaseNotes]]
```

Comment:
(soft) Maybe we want this release-notes.adoc to include a release-notes-8.adoc so it doesn't grow into a massive file?